muffnn/mlp/tests/test_mlp_classifier.py

```diff
@@ -299,3 +299,14 @@ def test_cross_val_predict():
     auc = roc_auc_score(y, y_oos, average=None)
 
     assert np.all(auc >= 0.96)
+
+def test_embedding():
+    # Check that fitting twice works (e.g., to make sure that fit-related
+    # variables are cleared appropriately when refitting).
+
+    X, y = iris.data, iris.target
+
```

Comment:
i think it'd be good to have a slightly more comprehensive test that at least makes sure that the `transform_layer_index` argument behaves as expected.